core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -322,28 +322,22 @@ private[history] class FsHistoryProvider(conf: SparkConf) extends ApplicationHis
    *
```

Comment:
This doc might be better if it said "in Spark 1.2.X and earlier" instead of "in previous releases".